src/addr2line.c

```diff
@@ -1,3 +1,4 @@
+// -*- mode: C; c-basic-offset: 8; tab-width: 8; indent-tabs-mode: t -*-
```

Comment:
do we need this line? SPDX header has to be the first line in the file, so we either need to move it down or just drop it